index.html

```diff
@@ -26,12 +26,112 @@
+                        <img src="images/icon.svg" height="12px" ></button>
+                </div>
+            </div>
+            <div class="grid-item">
+                <img class="card_image " src="images/4.png ">
+                <div class="card_content ">
+                    <div>
+                        <h2 class="card_title ">Project name goes here</h2>
+                    </div>
+                    <div class="span_button">
+                        <span href="" class="btn card_btn1 ">HTML/CSS</span>
+                        <span href="" class="btn card_btn2 ">Ruby on Rails</span>
+                        <span href="" class="btn card_btn3 ">JavaScript</span>
+                    </div>
+                    <button class="card_btn"><span>See this project</span>
+                        <img src="images/icon.svg" height="12px" ></button>
+                </div>
+            </div>
+            <div class="grid-item">
+                <img class="card_image " src="images/5.png ">
+                <div class="card_content ">
+                    <div>
+                        <h2 class="card_title ">Project name goes here</h2>
+                    </div>
+                    <div class="span_button">
+                        <span href="" class="btn card_btn1 ">HTML/CSS</span>
+                        <span href="" class="btn card_btn2 ">Ruby on Rails</span>
+                        <span href="" class="btn card_btn3 ">JavaScript</span>
+                    </div>
+                    <button class="card_btn"><span>See this project</span>
+                        <img src="images/icon.svg" height="12px" ></button>
+                </div>
+            </div>
+            <div class="grid-item">
+                <img class="card_image " src="images/6.png ">
+                <div class="card_content ">
+                    <div>
+                        <h2 class="card_title ">Project name goes here</h2>
+                    </div>
+                    <div class="span_button">
+                        <span href="" class="btn card_btn1 ">HTML/CSS</span>
+                        <span href="" class="btn card_btn2 ">Ruby on Rails</span>
+                        <span href="" class="btn card_btn3 ">JavaScript</span>
+                    </div>
+                    <button class="card_btn"><span>See this project</span>
+                        <img src="images/icon.svg" height="12px" ></button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="about">
+        <h1>About</h1>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+        <a href="" class="button"><span>See this project</span>
+        <img src="images/icon1.svg" height="16px" ></a>
     </div>
```

Comment:
- [ ] Kindly use a `section` instead of a `div` for this element.
- [ ] Please apply the appropriate `hx` element for the inside header instead of a `h1` tag.
- [ ] Kindly try matching the Figma design as there is more content underneath that describes skills.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/205421718-f2308f1e-1bd5-4fd6-a467-1baa0abbad05.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/205421762-41b92bb0-c0a8-48a8-96e1-681b2b7df55e.png" width="300">|